data/solutions-data.tsx

```diff
@@ -7,25 +7,86 @@ import moldovaSvg from '../images/moldova.svg'
 
 export const solutionsData = [
   {
+    slug: 'voluntar-md',
     title: 'voluntar.md',
+    longTitle: 'voluntar.md',
     website: 'https://voluntar.md/',
+    githubRepository: 'https://github.com/code4moldova/voluntar-web',
     description: 'Platforma care conectează persoanele care au nevoie de ajutor cu cele care doresc să ajute.',
+    longDescription:
+      '<p class="mb-2">Odată cu izbucnirea pandemiei, a crescut numărul de persoane aflate în situații vulnerabile. Echipa de voluntari “Inițiativa Împreună Împotriva COVID19” vin în ajutor acestor oameni cu produse alimentare conectându-i cu voluntarii care pot oferi livrare.</p>' +
+      '<p class="mb-2"><strong>Problema:</strong> Un număr considerabil de pesoane aflate în situații de criză (nu au acces la produse alimentare, de igienă și acces la îngrijire medicală, etc.) nu beneficiază de suport.</p>' +
+      '<p class="mb-2"><strong>Soluția:</strong> Platformă online care conectează persoanele aflate în situații de criză și societatea civilă care ajută cu produse alimentare, de igienă și transport.</p>' +
+      '<ul class="list-disc list-inside pl-5">' +
+      '<li>Toate solicitările telefonice a persoanelor vulnerabile sunt înregistrate pe această platformă</li>' +
+      '<li>Orice voluntar care este dispus să ajute cu livrarea pachetelor și oferirii de transport poate să se înregistreze pe platformă</li>' +
+      '<li>Voluntarul primește acces la platformă și poate prelua solicitări</li>' +
+      '<li>Progresul fiecărei sarcini și statutul solicitărilor pot fi ușor gestionate și monitorizate</li>' +
+      '</ul>',
     image: girl,
     icon: moldovaSvg,
+    status: 'finisat',
```

Comment:
Is not finished yet :slightly_smiling_face:

Reply:
Took from the mockups :) Will remove it.